iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/execution/executor/RegionWriteExecutor.java

```diff
@@ -1045,6 +1045,7 @@ private RegionExecutionResult executeCreateOrUpdateTableDevice(
     @Override
     public RegionExecutionResult visitPipeEnrichedWritePlanNode(
         final PipeEnrichedWritePlanNode node, final WritePlanNodeExecutionContext context) {
+      node.setOriginClusterId(node.getOriginClusterId());
```

Comment:
...?

Reply:
Seemingly some extra operations are needed to pass the "originClusterId" to the state machine....